sections/tox.zsh

```diff
@@ -0,0 +1,48 @@
+#
+# Tox
+#
+# tox is a generic virtualenv management and test command line tool
+# Link: https://tox.readthedocs.io/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+SPACESHIP_TOX_SHOW="${SPACESHIP_TOX_SHOW=true}"
+SPACESHIP_TOX_PREFIX="${SPACESHIP_TOX_PREFIX=""}"
+SPACESHIP_TOX_SUFFIX="${SPACESHIP_TOX_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_TOX_COLOR_FAIL="${SPACESHIP_TOX_COLOR_FAIL="red"}"
+SPACESHIP_TOX_COLOR_OK="${SPACESHIP_TOX_COLOR_OK="green"}"
+SPACESHIP_TOX_FAIL_ICON="${SPACESHIP_TOX_FAIL_ICON="\u2718 "}"
+SPACESHIP_TOX_OK_ICON="${SPACESHIP_TOX_OK_ICON="\u2714 "}"
```

Comment:
`SPACESHIP_TOX_SYMBOL_OK` for consistency with `SPACESHIP_TOX_COLOR_OK` and `SPACESHIP_TOX_SYMBOL` :)